src/transformState.js

```diff
@@ -1,11 +1,32 @@
-'use strict';
+"use strict";
 
 /**
  * @param {Object} state
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    switch (obj.type) {
+      case "addProperties":
+        for (const keys in obj.extraData) {
+          state[keys] = obj.extraData[keys];
+        }
+        break;
+
+      case "removeProperties":
+        for (const key of obj.keysToRemove) {
+          delete state[key];
+        }
+
+        break;
+
+      default:
```

Comment:
it's not a default case, it's a `clear`, change it

Reply:
i corrected it